### DISCUSSION TITLE
Allow deactivation of non-custom domains

### DISCUSSION THREAD

**Discussion by stefanschramek** [anonaddy/anonaddy#359] [Feature Requests]
Hi,
it would be nice to allow the deactivation of non-custom domain to only get the domains if would like to use when generating a new alias.
Today if you have more than 1 custom domain and want to switch to non default during "generate new alias" it can be tricky to select the correct domain.
Thanks
Stefan

**Comment by willbrowningme** [comment]
I recently added the option to choose a default alias domain in your account settings.

This means that your preferred domain will always be preselected when you are generating aliases on the site or using the browser extension.

Does this solve the issue for you?

**Comment by stefanschramek** [comment]
I already set the default domain - but I use more than 1 custom domain!
So that does not really solve it - I think it would be better to give the option to disable given domains!
It‘s definitely not a big issue - just a nice to have!
